tests/hartree-fock/hartree-fock++.cc

```diff
@@ -217,9 +221,15 @@ int main(int argc, char* argv[]) {
     const auto filename = (argc > 1) ? argv[1] : "h2o.xyz";
     const auto basisname = (argc > 2) ? argv[2] : "aug-cc-pVDZ";
     bool do_density_fitting = false;
+    double cholesky_threshold = 1e-4;
 #ifdef HAVE_DENSITY_FITTING
     do_density_fitting = (argc > 3);
     const auto dfbasisname = do_density_fitting ? argv[3] : "";
+    if ((strcmp(dfbasisname, "autodf") == 0) && argc > 4) {
```

Comment:
you could avoid the need for extra argument for the cholesky parameter if you encode it in the name, e.g. the meaning of `auto(1e-7)` is obvious